doc/src/manual/running-external-programs.md

```diff
@@ -60,6 +60,39 @@ julia> open(`less`, "w", STDOUT) do io
 3
 ```
 
+## Cmd Object
+Normally, to create a `Cmd` object in the first place, one uses backticks, e.g.
+
+```jldoctest
+julia> Cmd(`echo "Hello world"`, ignorestatus=true, detach=false)
+`echo 'Hello world'`
+```
+
+There are some keyword arguments available to customize the `Cmd` object's usage. You can set the working directory for the command by using the `dir` kwarg, e.g.
+
+```jldoctest
+julia> Cmd(`pwd`, dir = "..")
+setenv(`pwd`; dir="..")
+```
```

Comment:
Perhaps something along the lines of the following would be better? Thoughts?

## Execution Environment

You can control a `Cmd`'s execution environment by one of several means. First, constructing a `Cmd` by explicitly calling the `Cmd` type constructor (rather than implicitly using solely backticks as illustrated above) allows you to specify several aspects of the `Cmd`'s execution environment via keyword arguments. For example, the `dir` keyword provides control over the `Cmd`'s working directory,
```julia
julia> run(Cmd(`pwd`, dir = "/"))
/
```
and the `env` keyword allows you to set execution environment variables,
```julia
julia> run(Cmd(`bash -c "echo foo \$HOWLONG"`, env = ("HOWLONG" => "ever!",)))
foo ever!
```
See `[`Cmd`](@ref)` for additional keyword arguments. [`setenv`](@ref) provides another means for setting `Cmd` execution environment variables,
```julia
julia> run(setenv(`bash -c "echo foo \$HOWLONG"`, ("HOWLONG" => "ever!",)))
foo ever!
```
and similarly [`withenv`](@ref) for modifying rather than replacing the execution environment,
```julia
julia> withenv(() -> run(`bash -c "echo foo \$HOWLONG"`), "HOWLONG" => "ever!")
foo ever!
```
which commonly appears in `do`-block form,
```julia
julia> withenv("HOWLONG" => "ever!") do
           run(`bash -c "echo foo \$HOWLONG"`)
       end
foo ever!
```